xml/System.Data/DataRowView.xml

```diff
@@ -308,6 +308,11 @@
         <summary>Returns a <see cref="T:System.Data.DataView" /> for the child <see cref="T:System.Data.DataTable" /> with the specified <see cref="T:System.Data.DataRelation" /> and parent.</summary>
         <returns>A <see cref="T:System.Data.DataView" /> for the child <see cref="T:System.Data.DataTable" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException"><paramref name="relation" /> is <see langword="null" />.
+        
+-or- 
+
+There was a mismatch between <paramref name="relation" /> and <see cref="M:System.Data.DataView.get_Table" />.</exception>
```

Comment:
I think this was flagged by @mairaw before: there is no such thing as get_Table, and anyway I am not sure what this sentence means.

Reply:
So would you like me to remove the second sentence for this exception?

Reply:
I'll assume you want me to remove the second sentence, so I'll do it. If I was wrong, let me know and I'll bring it back so we can reword it.